util/mtt_update.py

```diff
@@ -0,0 +1,466 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Script to generate Minetest translation template files and update
+# translation files.
+#
+# Copyright (C) 2019 Joachim Stolberg, 2020 FaceDeer, 2020 Louis Royer,
+#                    2023 Wuzzy.
```

Comment:
Is this indentation intentional? It can be arranged into multiple lines (one line for each person):

```suggestion
# Copyright (C) 2019 Joachim Stolberg
# Copyright (C) 2020 FaceDeer
# Copyright (C) 2020 Louis Royer
# Copyright (C) 2023 Wuzzy
```


I am fine either way.